addons/website_blog/controllers/main.py

```diff
@@ -167,7 +175,7 @@ def blog(self, blog=None, tag=None, page=1, **opt):
         if blog:
             values['main_object'] = blog
             values['edit_in_backend'] = True
-            values['blog_url'] = QueryURL('', ['blog', 'tag'], blog=blog, tag=tag, date_begin=date_begin, date_end=date_end)
+            values['blog_url'] = QueryURL('', ['blog', 'tag'], blog=blog, date_begin=date_begin, date_end=date_end)
```

Comment:
why remove tag here?

Reply:
I removed it because otherwise blog_url() would always return the url with the tags for the tag cloud in a blog. Which make the queryUrl I use for the post request wrong